src/include/iscsi-lio-server/TargetData.rb

```diff
@@ -504,6 +491,27 @@ def get_target_list()
     list
   end
 
+private
+
+  # Saves acl authentication information
+  def save_current_acl_rule
+    @current_acl_rule = @current_acls_group.fetch_rule(@initiator_name)
+
+    @current_acl_rule.store_userid(acl_auth_info("userid"))
+    @current_acl_rule.store_password(acl_auth_info("password"))
+    @current_acl_rule.store_mutual_userid(acl_auth_info("mutual_userid"))
+    @current_acl_rule.store_mutual_password(acl_auth_info("mutual_password"))
+  end
+
+  # Runs a targetcli command to get specific acl authentication information
+  #
+  # @param data [String] e.g., "userid", "password", "mutual_userid" or "mutual_password"
+  # @return [String]
+  def acl_auth_info(data)
+    acl = "iscsi/#{@current_target.fetch_target_name}/tpg#{@current_tpg.fetch_tpg_number}/acls/#{@initiator_name}/"
+
+    Yast::Execute.locally!.stdout("targetcli", acl, "get", "auth", data).split("=").last
```

Comment:
Can the value in the output also contain `=` character? If yes then it's usually better to limit the split result. E.g. parsing

```
FOO="bar=baz"
```

works better with if the `#split` is limited to max. 2 values: `var, value = line.split("=", 2)`.